### ISSUE TITLE
webfonts-loader seemingly not adding @font-face css enties to Webpack bundle

### ISSUE THREAD

**Issue by FarizF** [jeerbl/webfonts-loader#103]
Hi!

Maybe this should not be on here, as I'm sure it's me doing something wrong, not the plugin. Still, I'm hoping you're willing to help you with my issue.

I'm trying to retrieve fonts from another npm package locally hosted on the company servers using webfonts-loader. Now, we have done this already once for another project, but I seem to miss some configuration that causes the `@font-face` rules to not be added to the bundle.css. In the other project, the font-face is correctly added, I just can't seem to find that piece of configuration whatsoever. I have tried add my font-family by adding the following webpack config entry and rule:

```
'app': [
    ....
    './node_modules/corporate-identity/src/icons/build-templates/company-icons.font.js',
],
```

```
{ test: /\.font\.js/, use: [MiniCssExtractPlugin.loader, 'css-loader', { loader: 'webfonts-loader', options: { publicPath: "/"} }]},
```

The contents of the font.js configuration retrieved from the node_modules folder file are as follows:

```
module.exports = {
    'files': [
        '../*.svg'
    ],
    'fontName': 'company-icons',
    'classPrefix': 'icon-',
    'baseSelector': '.icon',
    'types': ['eot', 'woff', 'woff2', 'ttf', 'svg'],
    'cssTemplate': 'stylesheet-template.css',
    'dest': 'assets/',
    'fileName': 'assets/[fontname].[hash].[ext]'
};
```

And the corresponding template:
```
@font-face {
    font-family: "{{fontName}}";
    src: {{{src}}};
    font-weight: normal;
    font-style: normal;
}

{{baseSelector}}:before {
    display: inline-block;
    font-family: "{{fontName}}";
    font-style: normal;
    font-weight: normal;
    line-height: 1;
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
}

{{baseSelector}}-lg {
    font-size: 1.3333333333333333em;
    line-height: 0.75em;
    vertical-align: -15%;
}
{{baseSelector}}-2x { font-size: 2em; }
{{baseSelector}}-3x { font-size: 3em; }
{{baseSelector}}-4x { font-size: 4em; }
{{baseSelector}}-5x { font-size: 5em; }
{{baseSelector}}-fw {
    width: 1.2857142857142858em;
    text-align: center;
}


{{#each codepoints}}
.{{../classPrefix}}{{@key}}:before {
    content: "\\{{this}}";
}
{{/each}}
```

In short:
**Which piece of missing configuration is causing the font-face entries to not be added?**

**Comment by FarizF** [comment]
My apologies. It's been a long day 😅 

I didn't think it was necessary to the `require` the font config as it wasn't present in the angular entry file of the project that did have the correct `@font-face`  entries. Works after adding the require.
